### PR TITLE
Create Measurementmanager interface

### PR DESCRIPTION
**Pull request by vamossagar12**
As requested here:
[https://github.com/kubernetes/perf-tests/pull/1075/files#r383722052](url) , creating a separate PR for creating a measurementmanager interface which would and adding Measurementmanager to measurementConfig.

### CODE REVIEW

File: clusterloader2/pkg/measurement/interface.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ type MeasurementConfig struct {
 	// TemplateProvider provides templated objects.
 	TemplateProvider    *config.TemplateProvider
 	ClusterLoaderConfig *config.ClusterLoaderConfig
+	MeasurementManager  MeasurementManager
```
[the remark this answers is not in the document]
Comment (vamossagar12) [reply clusterloader2/pkg/measurement/interface.go]: Actually, that change will be eventually needed for this issue: https://github.com/kubernetes/perf-tests/issues/794 as we had discussed here:
https://github.com/kubernetes/perf-tests/issues/794#issuecomment-588870480. 

Should I remove it from this PR and add it separately in the other PR?

File: clusterloader2/pkg/measurement/manager.go [hunk 2]
```diff
@@ -70,20 +77,20 @@ func (mm *MeasurementManager) Execute(methodName string, identifier string, para
...
+func (mm *measurementManager) Dispose() {
 	for _, instances := range mm.measurements {
 		for _, measurement := range instances {
 			measurement.Dispose()
 		}
 	}
 }
 
-func (mm *MeasurementManager) getMeasurementInstance(methodName string, identifier string) (Measurement, error) {
+func (mm *measurementManager) GetMeasurementInstance(methodName string, identifier string) (Measurement, error) {
```
[the remark this answers is not in the document]
Comment (vamossagar12) [reply clusterloader2/pkg/measurement/manager.go]: Same comment as below.

File: clusterloader2/pkg/measurement/interface.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (mm4tt) [reply clusterloader2/pkg/measurement/interface.go]: Yes

[the remark this answers is not in the document]
Comment (vamossagar12) [reply clusterloader2/pkg/measurement/interface.go]: ok. Done

**Comment by mm4tt** [comment]
Thanks @vamossagar12!

**Comment by vamossagar12** [comment]
thanks. I will integrate these changes in the wrapper measurements PR.

**Pull request merged**